src/class-parsely.php

```diff
@@ -555,18 +555,39 @@ public function get_options() {
 		 */
 		$options = get_option( self::OPTIONS_KEY, null );
 
-		// @phpstan-ignore isset.offset, booleanAnd.alwaysFalse
+		// Existing plugin installation without full metadata option.
+		/* @phpstan-ignore isset.offset, booleanAnd.alwaysFalse */
 		if ( is_array( $options ) && ! isset( $options['full_metadata_in_non_posts'] ) ) {
-			// Existing plugin installation without full metadata option.
 			$this->set_default_full_metadata_in_non_posts();
 		}
 
-		// @phpstan-ignore isset.offset, booleanAnd.alwaysFalse
+		// Existing plugin installation without Content Helper options.
+		/* @phpstan-ignore isset.offset, booleanAnd.alwaysFalse */
 		if ( is_array( $options ) && ! isset( $options['content_helper'] ) ) {
-			// Existing plugin installation without Content Helper options.
 			$this->set_default_content_helper_settings_values();
 		}
 
+		// Existing plugin installation that's missing a Content Helper feature option.
+		/* @phpstan-ignore isset.offset */
+		if ( is_array( $options ) && isset( $options['content_helper'] ) ) {
+			/** @var array<string,Parsely_Options_Content_Helper_Feature> $pch_options */
+			$pch_options = $options['content_helper'];
+
+			/** @var array<string,Parsely_Options_Content_Helper_Feature> $pch_options_defaults */
+			$pch_options_defaults = $this->option_defaults['content_helper'];
+
+			if ( count( $pch_options ) !== count( $pch_options_defaults ) ) {
```

Comment:
I could see cases where an option is removed from defaults where it might fail, or an option is removed but another is added and the count stays the same. Would it be possible to use something like `array_replace_recursive()` to auto-merge defaults? e.g.

```php
$pch_options_defaults = $this->option_defaults['content_helper'];
$options = array_replace_recursive( $pch_options_defaults, $pch_options );
```